tendrl/monitoring_integration/alert/handlers/cluster/gluster/cluster_utilization_handler.py

```diff
@@ -38,13 +38,13 @@ def format_alert(self, alert_json):
             if alert['severity'] == "WARNING":
                 alert['tags']['message'] = ("Cluster utilization of cluster %s is" \
                 " %s which is above the %s threshold (%s)." % (
-                    alert['tags']['cluster_name'],
+                    alert['tags']['integration_id'],
                     alert['current_value'],
                     alert['severity'],
                     alert['tags']['warning_max']
                 ))
             elif alert['severity'] == "INFO":
-                alert['tags']['message'] = ("Cluster utilization of cluster %s is"\
+                alert['tags']['integration_id'] = ("Cluster utilization of cluster %s is"\
                 " back to normal" % (
                     alert['tags']['cluster_name']
```

Comment:
@GowthamShanmugam I feel there is a requirement to replace cluster_name over here as well

Reply:
ya i have wrongly renamed